frontend/test/metabase/scenarios/dashboard/parameters.cy.spec.js

```diff
@@ -130,6 +132,7 @@ describe("scenarios > dashboard > parameters", () => {
     cy.get(".Dashboard .Icon-gear").click();
     cy.findByText("Remove").click();
     cy.findByText("Save").click();
+    cy.findByText("You're editing this dashboard.").should("not.exist");
```

Comment:
This is an additional safeguard which doesn't affect the outcome of this test. We're simply waiting for the dashboard to be fully saved before asserting on the URL change.